lte/gateway/configs/sessiond.yml

```diff
@@ -52,7 +52,7 @@ sessions_table: sessiond:sessions
 
 # Set to true if converged access set message support is required or 5g access
 # support is required
-converged_access: true
+enable5g_features: true
```

Comment:
why is this true by default?

Reply:
Yes Ideally default value should be false. User can manually modify to true if they want 5G call. I modified the default value as false.